.gitignore

```diff
@@ -2,6 +2,8 @@
 /typings/
 /dist/
 /temp/
+/demo/dist/
```

Comment:
Dist folder is something we have to talk about yet. We are not really sure in how are we going to deploy it.

On the other hand, the demo deploy is just a temporary folder. I could even create a task to clean it after success push.

So the idea is, when you need to update the gh-pages, you run a command, `/demo/dist` gets generated and then pushed to gh-pages. After that you don't need and if we decide to put root's `/dist` on NPM, I wouldn't certainly want the demo dist in there.

So it is a -1 for me.